### ISSUE TITLE
Could the studio interface add a folder creation feature to help categorize and organize when there are too many applications?

### ISSUE THREAD

**Issue by Moiyu1998** [langgenius/dify#10349]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I've created a lot of applications, and it's easy to get them mixed up when there are too many. Could a folder feature be added to help with categorizing and organizing the applications?

### 3. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by crazywoola** [comment]
Related issues #7966 #3619

**Comment by Moiyu1998** [comment]
> Hi, @Moiyu1998. I'm [Dosu](https://dosu.dev), and I'm helping the Dify team manage their backlog. I'm marking this issue as stale.
> 
> **Issue Summary:**
> 
> * You requested a feature to create folders in the studio interface for better organization of multiple applications.
> * This feature aims to address the difficulty in managing numerous applications.
> * User crazywoola mentioned related issues, [Request for Folder Management and Batch Export Features #7966](https://github.com/langgenius/dify/issues/7966) and [Folder Organization Feature for Projects in Studio Page #3619](https://github.com/langgenius/dify/discussions/3619), which might offer additional context or similar requests.
> 
> **Next Steps:**
> 
> * Please let us know if this issue is still relevant to the latest version of the Dify repository by commenting here to keep the discussion open.
> * If there is no further input, this issue will be automatically closed in 15 days.
> 
> Thank you for your understanding and contribution!

Yes, I still hope you can add this feature, which is very practical.
